spec/helpers/tweets_helper_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+describe TweetsHelper, :vcr do
+
+  describe 'Favorite button' do
+
+    it 'returns the html for the button' do
+      tweet = TwitterClient.new.search('463440424141459456').first
+
+      expect(favorite_button(tweet)).to have_css(:span)
```

Comment:
Pude testear que sea una estrella.
Quería testear también que haya un botón con la clase correcta, pero vi que no se ponen más de un **expect** por test. Poner dos tests solo para eso me parece mucho, hay otra forma de hacer un expect con varias cosas? Por ej: un tipo de **have_css AND have_button..**